src/utils/importFuncGenerator.ts

```diff
@@ -116,14 +109,11 @@ export function createImport(
     case encodePath(UINT256_LT):
     case encodePath(UINT256_MUL):
     case encodePath(UINT256_SUB):
-      return createFuncImport('range_check_ptr');
     case encodePath(DEPLOY):
     case encodePath(EMIT_EVENT):
     case encodePath(GET_CALLER_ADDRESS):
     case encodePath(GET_CONTRACT_ADDRESS):
-      return createFuncImport('syscall_ptr');
-    // Import libraries from Cairo1
-    case encodePath(INTO):
+    case encodePath(INTO): // Import libraries from Cairo1
     case encodePath(ADDRESS_INTO_FELT):
     case encodePath(U128_TO_FELT):
     case encodePath(U256_FROM_FELTS):
```

Comment:
I think we could remove the whole switch since now there are no other implicits than `warp_memory`